packages/editor-ui/src/stores/workflows.store.spec.ts

```diff
@@ -0,0 +1,444 @@
+import { setActivePinia, createPinia } from 'pinia';
```

Comment:
are we not keeping frontend tests in `__tests__` folders anymore?

Reply:
We agreed at the last Front End team meeting to put tests alongside source files to promote testing. We'll be moving the other tests to follow the same pattern soon.